### ISSUE TITLE
problem parsing <pb/> tag

### ISSUE THREAD

**Issue by shadoof** [shadoof/rtwtr#32]
The tag should render (as a paragraph break, typically with blank line, as for `<p>` in html) where it occurs in the A and B files.

Bug is visible in section 2. The paragraph break should come between "... typographic visuality." and "Integrated in ...". Instead it rendered after the sentence, after "... 'historical consciousness.'"

Note also that if the `<pb/>` tag is move _within_ the preceding `<ub>` unit, no paragraph break is rendered at all. These `<pb/>` should be able to be rendered _within_ units.

**Comment by cqx931** [comment]
The bug should be fixed in this commit.
https://github.com/shadoof/rtwtr/pull/33/commits/127961af760329eeb1665de46582c93e4ef706ab

However, supporting <pb/> tag within <ub> might still cause trouble in the current commit, I will revisit this later

**Comment by cqx931** [comment]
I can work on this so that pb tag can be included within <ub>.
However, there can't be only part of a paragraph that belongs to a unit, it has to be one full paragraph.

See: the following example in a pseudo dom structure. See p#2 in <unit> can't be the same paragraph as p#3. This is beyond the capacity of the current unit structure if we try to keep one unit in the same dom element.

<unit>
  <p id="1">
   t
   t
  <p>
  <p id="2">
    t
    t
  </p>
</unit>

<p id="3">
  t
  t
  t
</p>

**Comment by shadoof** [comment]
Right. You fixed this for Section 2 in our current text. But I haven't tried to follow how you did this, so apologies if we are discussing at cross purposes.

The html `<p>[</p>]` tag-pair [or tag] has always seemed to me to be a terrible anomaly and if _our_ `<pb/>` _**had**_ to be rendered in terms of an html `<p>` then I can see that this makes it, basically, impossible for part of a `<p>` paragraph to be in one unit, with any other paragraph part in another unit.

So, in that case, our `<pb/>` should just be rendering convention, and nothing to do with the DOM structurally. It should just be translated to whatever the webapp's style definition of a typographic paragraph break may be: in our case: two linefeeds and a carriage return, or something like `<br/>&nbsp;<br/>`.

I hope I've got this right and that it makes sense to you. Bottom line: don't spend a lot of time on this, especially since you've solved it for section 2.

**Comment by cqx931** [comment]
Yes that's correct. Though I'm not sure whether it is the right thing to do to use 
the `<br/>&nbsp;<br/>` hack to replace `<p>`. I will do it the other way around by defining units through numbered class names instead of using a real span wrapper. If that's not really the priority at the moment, let's remove the "needed for DR" label.

**Comment by shadoof** [comment]
Agreed. That label removed.
